<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1291</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Box-average smoothing</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonInFile">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Input file</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInFile">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>1201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelOutFile">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>1201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOutFile">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Output file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSmooth">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Smooth</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Result:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelResult">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>370</y>
      <width>1231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelSize">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string> R</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSize">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelNCPU">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>NCPU</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNCPU">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>381</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>The averaging cube is NxNxN where N = 2R+1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>NCPU is the number of threads to use on a multiprocessor computer</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>60</y>
      <width>451</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1291</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonInFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>inFileSelecter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonOutFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>outFileSelecter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>18</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>58</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSmooth</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>smoother()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>338</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>buttonClickHandler()</slot>
  <slot>inFileSelecter()</slot>
  <slot>outFileSelecter()</slot>
  <slot>smoother()</slot>
 </slots>
</ui>
